travis/install_custom_pg

```diff
@@ -15,8 +15,14 @@ pkgingurl='https://anonscm.debian.org/git/pkg-postgresql/postgresql.git'
 patchurl="${pkgingurl}/plain/debian/patches/50-per-version-dirs.patch"
 
 # clone PostgreSQL
+if [ "${PGVERSION}" -eq '10' ]; then
+    gitref='REL_10_BETA1'
+else
+    gitref="REL${PGVERSION//./_}_STABLE"
+fi
+
```

Comment:
Hm. Good point. For some reason I figured features could keep going in master while the release branch diverged, but I guess the whole point is there is no release branch yet and the beta tag lives on master instead. Will just change to master.